run_it.sh

```diff
@@ -10,4 +12,4 @@ docker build . -t "openbrewerydb:$tag"
 export TAG=$tag
 
 # Deploy to docker-compose
-docker-compose up -d
+docker-compose up
```

Comment:
Please re-add `-d` as it is a much more sensible default for a convenience script.

Reply:
+1 https://github.com/openbrewerydb/openbrewerydb-java-api/pull/20/commits/55aaaf75a34d94f523973425b90a7420f0c6c70e